src/sql/workbench/parts/notebook/browser/notebook.component.ts

```diff
@@ -427,6 +427,8 @@ export class NotebookComponent extends AngularDisposable implements OnInit, OnDe
 		this._trustedAction = this.instantiationService.createInstance(TrustedAction, 'notebook.Trusted');
 		this._trustedAction.enabled = false;
 
+		let collapseCellsAction = this.instantiationService.createInstance(CollapseCellsAction, 'notebook.collapseCells');
```

Comment:
can we reference a constant instead of hardcoding the name here? just in case it gets changed in the future

Reply:
Probably a broader effort, since we just have random strings floating around for a lot of these createInstance calls.